wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss

```diff
@@ -5,7 +5,6 @@
 		}
 	}
 
-	&.course:not(.post-type-archive-course) article.course:first-child,
```

Comment:
Not entirely sure what this is supposed to target, but came from the customiser and ends up breaking the first card in the search results